python/CMakeLists.txt

```diff
@@ -118,7 +122,7 @@ if (NOT MSVC)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-omit-frame-pointer")
 
   # Suppress Cython warnings
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-variable -Wno-maybe-uninitialized")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-variable -Wno-maybe-uninitialized -Wno-conversion")
```

Comment:
This change doesn't affect MinGW build. Because it's C++ build not Python build.

Reply:
Bad change that went in while trying to figure out the issue.